utils/mock-data-loader/helpers/generate-salt-and-hash.ts

```diff
@@ -0,0 +1,12 @@
+import crypto from 'crypto';
```

Comment:
* Please port to `libs/common`.
* DocString please.
* Unit test cases

Reply:
Commonising the password hashing is beyond the scope of what I am doing here so I think, if we want to do that, we should do so with an actual ticket

Reply:
I think we should look into the scope of this PR in that case as I am trying to understand if the whole mock data loader refactor is required at this instance.

Reply:
I'd really like to talk about JSDocs in dev friday. I wrote a whole thing about how we don't really need them here -- but looking at this it's probably good to document it's coming out as a `hex` string, so that could help!